src/common/tiles/BigTile/index.js

```diff
@@ -37,16 +38,32 @@ export const BigTile = ({
   genres,
   profile_path,
 }) => {
+  const dateOfRelase = new Date(release_date);
+  const dayOfBirth = new Date(birthday);
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    window.addEventListener("resize", setMobileState);
+
+    return () => {
+      window.removeEventListener("resize", setMobileState);
+    }
+  }, []);
+
+  const setMobileState = () => {
```

Comment:
uważam, że to powinno się znaleźć nad useEffect, jak analizowałam kod troche mnie to zmyliło, bo nie widziałam co to jest setMobileState ( najpierw definiujemy zmienne, a później z nich korzystamy )